specification/sql/resource-manager/Microsoft.Sql/preview/2023-08-01-preview/LocationCapabilities.json

```diff
@@ -419,6 +422,14 @@
           "description": "Supported time range for per database auto pause delay",
           "readOnly": true
         },
+        "supportedZones": {
```

Comment:
Should this be renamed to something like "supportedZoneCapabilities" or similar?

Reply:
these are specific SQL DB terms